test/claim.test.ts

```diff
@@ -119,4 +131,81 @@ describe('StakingPool.claim', () => {
     });
   });
 
+
+  context('claim after pool is closed', async () => {
+    beforeEach('deploy staking pool and init the pool', async () => {
+      await testEnv.rewardAsset.connect(deployer).faucet();
+      await testEnv.rewardAsset.connect(deployer).approve(testEnv.stakingPool.address, RAY);
+      await testEnv.stakingPool
+        .connect(deployer)
+        .initNewPool(
+          firstRoundInit.rewardPersecond,
+          firstRoundStartTimestamp,
+          firstRoundInit.duration
+        );
+      await testEnv.stakingAsset.connect(alice).faucet();
+      const tx = await testEnv.stakingAsset.connect(alice).approve(testEnv.stakingPool.address, RAY);
+      await advanceTimeTo(await getTimestamp(tx), firstRoundStartTimestamp);
+      await testEnv.stakingPool.connect(alice).stake(amount);
+    });
+
+    it('pool is closed and alice claim reward', async () => {
```

Comment:
I think `pool is closed` is redundant. How about wrapping the statement like below?
```
context('the pool is closed', async () => {})

or

beforeEach('the pool is closed', async () => {}) ~~
```